google-cloud-storage/.rubocop.yml

```diff
@@ -1,3 +1,5 @@
+require: rubocop-minitest
```

Comment:
Can we remove this now that changes have been made and look good?

Reply:
Sure. I have reverted this change.